### PR TITLE
Remove inputs and simplify interface

### PR DESCRIPTION
**Pull request by nikever**
This PR simplifies the interface of the module.
Now only the `furyagent_bucket_name` is required as input, and new optional variable `tags` is added to replace the removed variables.

PR Review:
- removes the input variables: `cluster_name`, `environment` that were mainly used to add fixed tags. Now tags can be consistent with other resources. 
- uses the `furyagent_bucket_name` to name IAM policy and users (bucket name is already unique by default). 
- removes the input variables: `aws_region` which is now taken from the bucket region. This is consistent as the bucket is created in the AWS provider current region.

### CODE REVIEW

**Comment by angelbarrera92** [comment]
Ciao @nikever ! My bigger concern here is how to upgrade from the current release. Is it possible without recreating the resources?
Maybe it could be a good idea to write down the changes from a terraform project perspective :) 
Don't forget to create the `docs/releases/unreleased.md` file :)
Anyway, LGTM!

**Review by angelbarrera92 — APPROVED** [review]
LGTM!

**Review by nutellinoit — APPROVED** [review]
LGTM 🎉

**Comment by nikever** [comment]
I added the documentation and tested it locally. Everything should be set.

File: docs/releases/unreleased.md [hunk 1]
```diff
@@ -19,3 +22,31 @@ $ furyagent <command>
 ```
 
 [`furyagent`]: https://github.com/sighupio/furyagent
+
+### Change the interface of terraform modules
```
Comment (angelbarrera92) [inline docs/releases/unreleased.md:26]: Thanks for clarifying the change :)

**Review by angelbarrera92 — APPROVED** [review]
LGTM!

**Pull request merged**